plugin/openassetio_manager_bal/BasicAssetLibraryInterface.py

```diff
@@ -30,6 +30,7 @@
 
 from . import bal
 
+
```

Comment:
[question][minor] There's a couple of whitespace changes in here, conflicting styles in the file?

Reply:
It's PyCharm's auto-formatter conforming to pseudo-PEP8. 

From [the PEP8 docs](https://peps.python.org/pep-0008/#blank-lines)

> Surround top-level function and class definitions with two blank lines.

To be fair, that doesn't specifically apply here, since `__all__` is not a class definition.  Regardless, I think it's pretty standard to have two blank lines after `import`s.

`pylint` doesn't catch (most) PEP8 violations, and `black` doesn't seem to care much about PEP8 compliance either.  Plus I'm sure PyCharm has opinions that are not strictly PEP8, like the above example.

It would be good to integrate `flake8` checks, like we do in other projects.

Anyway, looking at the whitespace changes in this PR, I'm happy to remove them if you prefer.

Reply:
I don't mind, so long as this is the correct way and it dosen't keep going back and forth.